### PR TITLE
v0.21.0 docs update 

### PR DESCRIPTION
**Pull request by toomuchdesign**
## What kind of change does this PR introduce?

Docs update

### CODE REVIEW

**Comment by toomuchdesign** [comment]
@all-contributors please add @jesstelford for bug

**Comment by toomuchdesign** [comment]
@all-contributors please add @massimeddu-sonic for bug

**Comment by toomuchdesign** [comment]
@all-contributors please add @fracture91 for bug

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.21.0
+
+### Breaking Changes
+
+- `initTestHelpers()` setup has to be removed with Jest/Jasmine-like test runners
```
Comment (Meemaw) [inline CHANGELOG.md:7]: I'm not quite understanding this change -- we are still exporting `initTestHelpers` and if users are using it there is no harm.

Comment (toomuchdesign) [reply CHANGELOG.md:7]: No big harm, I just wanted to highlight that the setup routine has changed. I should rephrase it.

**Review by Meemaw — APPROVED** [review]

**Pull request merged**